config/locales/en/layouts.yml

```diff
@@ -7,6 +7,7 @@ en:
       cookies: Cookies
       privacy: Privacy
       terms_and_conditions: Terms and conditions
+      accessibility: Accessibility statement
```

Comment:
I would name this just 'Accessibility' for short, in line with other services I've seen, like the Claim for costs of a child’s funeral.